Trabajo en el login

GabMesa opened the pull request:
Agregar al Login , el abrir una nueva pestaña llamada Root_Frame cuando
se ingresa el usuario Root con contraseña Rootlindo.
Se elimina la VentanaPrincipal
Diferenciacion en el login entre usuario o contraseña incorrecta.
10/09/16 (Gabriel)
Ps : avisar si llega el mensaje y correjir faltas de ortografia

UsernameTaken420 commented:
No estoy de acuerdo con ponerle la imagen de la ferretería como ícono, de todos modos se supone que eso lo vemos despues

GabMesa committed:
Trabajo en el login

Agregar al Login , el abrir una nueva pestaña llamada Root_Frame cuando
se ingresa el usuario Root con contraseña Rootlindo.
Se elimina la VentanaPrincipal
Diferenciacion en el login entre usuario o contraseña incorrecta.
10/09/16 (Gabriel)
Ps : avisar si llega el mensaje y correjir faltas de ortografia

GabMesa committed:
Agregar imagen Ferreteria a login

En icono y en ventana FotoFerreteria
